ydb/core/kqp/ut/view/view_ut.cpp

```diff
@@ -408,6 +408,46 @@ Y_UNIT_TEST_SUITE(TSelectFromViewTest) {
         CompareResults(etalonResults, selectFromViewResults);
     }
 
+    Y_UNIT_TEST(OneTableUsingRelativeName) {
```

Comment:
I would like to write an extensive test suite for views after this PR, so I add only a basic test of the new feature.